### PR TITLE
Xcode 7.1 recommended settings

### PR DESCRIPTION
**Pull request by mdiep**

### CODE REVIEW

File: Carthage.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -728,11 +728,11 @@
 			isa = XCBuildConfiguration;
 			baseConfigurationReference = D0D1212619E878CC005E4BAA /* Debug.xcconfig */;
 			buildSettings = {
+				ENABLE_TESTABILITY = YES;
 				GCC_NO_COMMON_BLOCKS = YES;
 				MACOSX_DEPLOYMENT_TARGET = 10.9;
 				ONLY_ACTIVE_ARCH = YES;
-				SWIFT_OPTIMIZATION_LEVEL = "-Onone";
-				SWIFT_WHOLE_MODULE_OPTIMIZATION = YES;
+				SWIFT_OPTIMIZATION_LEVEL = "-Owholemodule";
```
Comment (NachoSoto) [inline Carthage.xcodeproj/project.pbxproj]: Can you just remove this? It should be set from the `xcconfigs`  :)

**Comment by mdiep** [comment]
:sparkler:

**Comment by NachoSoto** [comment]
Looks like tests are currently failing, but unrelated to this!

**Comment by mdiep** [comment]
> Looks like tests are currently failing, but unrelated to this!

Weird. :confused:

**Pull request merged**